### PR TITLE
Update readme and plugin description

### PR DESCRIPTION
**Pull request by deepaklalwani97**
- Add izooto plugin requirement details.
- Add PWA plugin URL to the plugin description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,13 +37,15 @@ e.g Enable offline form submission with following settings for the forms availab
 
 ## Integration with izooto plugin.
 
+- The [iZooto](https://wordpress.org/plugins/izooto-web-push/) plugin must be installed and activated to use the izooto notification services.
```
Comment (DevikVekariya) [inline README.md:40]: This has to be added to - https://github.com/rtCamp/pwa-extension#requirements

add in the bracket that push notifications are dependent on that.

Comment (deepaklalwani97) [reply README.md:40]: Yes, it makes sense to keep it in both places.

File: README.md [hunk 2]
```diff
@@ -37,13 +37,15 @@ e.g Enable offline form submission with following settings for the forms availab
...
 - Registers izooto service worker using `wp_front_service_worker` hook to main service worker file
 - Removes Service worker code of izooto from `?izooto=sw` to prevent conflicting it with main service worker.
```
Comment (DevikVekariya) [inline README.md:42]: Is there any more descriptive steps needed to set up the iZooto?

Comment (deepaklalwani97) [reply README.md:42]: No, the user just needs to login with their izooto account with one click from izooto settings and details for which are already there in that setting page.

**Review by sagarnasit — APPROVED** [review]

**Pull request merged**